src/gsad_gmp.c

```diff
@@ -11984,7 +11984,9 @@ save_my_settings_gmp (gvm_connection_t *connection, credentials_t *credentials,
 {
   const char *lang, *text, *old_passwd, *passwd, *max;
   const char *details_fname, *list_fname, *report_fname;
+  const char *time_format, *date_format;
   gchar *lang_64, *text_64, *max_64, *fname_64;
+  gchar *time_format_64, *date_format_64;
```

Comment:
I think these would be better in the block that uses them. Most of the other, like `text_64`, are here because they are used by multiple cases.